### PR TITLE
VZ-5339: Enable access logs for Authproxy

### PR DESCRIPTION
**Pull request by adiforluls**
This change is to enable access logs for Authproxy.
Fixes VZ-5339

### CODE REVIEW

File: tests/e2e/logging/system/system_logging_test.go [hunk 1]
```diff
@@ -301,7 +301,7 @@ func validateAuthProxyLogs() bool {
 	}
 	exceptions = append(exceptions, istioExceptions...)
 	return validateElasticsearchRecords(
-		allElasticsearchRecordValidator,
+		noLevelElasticsearchRecordValidator,
```
Comment (kminder) [inline tests/e2e/logging/system/system_logging_test.go:304]: Why was this change required?

Comment (adiforluls) [reply tests/e2e/logging/system/system_logging_test.go:304]: `error_log` in NGINX records log level but not `access_log` it seems going by their docs (they have examples of specifying log severity level for error logs but not access logs).
